_data-prepper/expression-syntax.md

```diff
@@ -81,9 +86,9 @@ Used to test whether two values are equivalent.
 3.14 != /status_code
 {1, 2} == /event/set_property
 ```
-## Using equality operators to check JSON pointer 
+## Using equality operators to check for JSON Pointer 
 
-Equality operators can also be used to check whether a JSON Pointer exists by comparing with `null`.
+Equality operators can also be used to check whether a JSON Pointer exists by comparing the JSON Pointer with `null`.
```

Comment:
This is slightly confusing. How can we use something for comparison if we don't know it exists yet?